core/src/main/scala/cromwell/util/JsonEditor.scala

```diff
@@ -204,4 +205,39 @@ object JsonEditor {
     }
     workflowWithUpdatedCalls
   }
+
+  def extractSubWorkflowsMetadata(subworkflowId: String, workflowJson: Json): ErrorOr[Option[Json]] = {
```

Comment:
since the metadata for at most one subworkflow would be returned 
```suggestion
  def extractSubWorkflowMetadata(subworkflowId: String, workflowJson: Json): ErrorOr[Option[Json]] = {
```